media/css/products/monitor/common.scss

```diff
@@ -0,0 +1,242 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@use '~@mozilla-protocol/core/protocol/css/includes/lib' as * with ($font-path: '/media/protocol/fonts', $image-path: '/media/protocol/img');
+@use 'includes/tokens';
+
+main {
+    h1, h2, h3, h4, h5, h6 {
+        @include font-base;
+        line-height: tokens.$monitor-title-line-height;
+
+        &.t-breaches strong {
+            color: tokens.$monitor-color-purple;
+        }
+    }
+
+    .u-title-2xl {
+        @include font-size(tokens.$monitor-title-xl-size);
+        line-height: tokens.$monitor-title-line-height;
+
+        @media #{$mq-md} {
+            @include font-size(tokens.$monitor-title-2xl-size);
+        }
+    }
+
+    .u-title-xl {
```

Comment:
Nice work with this, if you ever wanted to update these values through the CSS variables you could do this:

```scss
:root {
  --title-sm-size: 1rem;
}
```
or however you'd like it, and that would update the values for all of the protocol helpers.